glossary.md

```diff
@@ -0,0 +1,3 @@
+* GBFS: [General Bikeshare Feed Specification](https://github.com/NABSA/gbfs)
+* GTFS: [General Transit Feed Specification](http://gtfs.org)
+* udata: in this codebase, refers to the Data Gouv API
```

Comment:
maybe you can add the udata repository: https://github.com/opendatateam/udata/

Reply:
C'est fait merci !